spec/models/report_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  describe '#editable?' do
+    before do
+      @alice = create(:user, name: 'アリス')
```

Comment:
テストコード中に"アリス"という文字列を使うコードはないので
```suggestion
      @alice = create(:user)
```

だけで問題ありません

Reply:
テストコードに出てこないカラムのデータはFactoryBotで定義したデフォルト値で問題ないということですね。
理解しました、こちら修正しました。